### ISSUE TITLE
[02/16] today I read

### ISSUE THREAD

**Issue by nmhkahn** [nmhkahn/paper-notes#3]
## Image Generation From Small Datasets via Batch Statistics Adaptation
- (이미지가 많은) 다른 데이터에서 pretrain 된 generator의 scale, shift 파라미터를 adapt해서 적은 이미지만 존재하는 데이터에 학습
    - 네트워크의 activation에 scale, shift를 적용하는 것은 activation의 strength나 (scale), threshold (shift)를 변경하는 것과 동일한 효과라는 가정하에서 출발
    - Class-condition BN을 사용하는 SNGAN에서 분석한 결과 gamma, beta가 class 특성에 맞는 정보를 잘 배운다고 함
-  학습은 gamma (scale), beta (shift)만 하며, 나머지는 네트워크 파라미터는 fix
    - Overfit을 막기위해 gamma, beta에 reg loss를 추가함
    -  데이터가 적어서 latent distribution을 densly 채우기 어려우니, z도 같이 optimize 하여 z에 의해 생성된 이미지가 학습 데이터에 있는 이미지와 유사하도록 가이드 (generative latent optimiation과 비슷하다고 함)

## Contrastive Learning for Unpaired Image-to-Image Translation
- Unpaired im2im에서 사용하는 cycle loss는 두 도메인간의 이미지가 bijection이라는 가정에 기반하므로 너무 strict하며, 이를 contrast loss를 사용해서 완화함
- Contrastive learning은 이미지 patch단위로 하는데, patch를 geneartor의 encoder와 MLP에 통과시킨뒤 각 patch들에 대해 similarity를 계산하고 N (negative) + 1 (positive) classificiation 수행
- Ablation study를 보면:
    - Negative patch를 늘리는게 당연히 좋고...
    - Identity loss (Y domain에 대해서도 patch_nce 계산) 사용해야하고
    - External patch보다 internal patch만 사용하는게 효과적이라고 함 <- MoCo 기반인거 같은데, 방법의 문제일까? 어쨌든 internal patch만 사용해도 strong하긴 함

## Few-Shot Unsupervised Image-to-Image Translation
- 기존 unsupervised (정확히는 unpaired...) im2im은 각 도메인에 대해 많은 데이터가 필요했으며, 이는 데이터가 얼마 없는 unseen domain에 일반화하기 어려움
- 그래서 few-shot unsupervised (unpaired) 문제를 제안하는데, 이 때 source 도메인은 다량의 데이터가 있는 multi-domain 이고, target 도메인은 이미지가 적은 시나리오를 가정함
- FUNIT 모델은 학습시에는 랜덤하게 두 도메인을 뽑고 content, style 이미지를 각각 랜덤하게 골라서 진행하며, 테스트시에는 기존 source 도메인에서 content image를, target 도메인에서 style image를 가져옴
    - 모델은 간단한데, content 이미지에서 spatial 정보를 encode 한 뒤 이를 G에 통과시키고, style 이미지는 vector로 encode해서 AdaIN에 넣음
    - 그래서 spatial 정보 (e.g. pose)는 content에서 가져오고, global한 texture 정보만 style에서 뽑도록 가이드함
    - Style 이미지가 여러장이라면 (few-shot) style vector를 mean <- 잘 되긴 하는데 이래도 되나..?

## COCO-FUNIT: Few-Shot Unsupervised Image Translation with a Content Conditioned Style Encoder
- Content loss: FUNIT은 content에 대한 제약이 없기 때문에 생성된 이미지의 content가 content image와 잘 align이 맞지 않는 경우가 생김
- COCO-FUNIT
    - Style vector를 뽑을 때 content image도 고려
    - Consant style bias를 style vector에 concat하는데, style 이미지에 over-sensitive하는걸 방지하는 역할을 한다고 함 (하이퍼파라미터로 조절가능) 

## High-Fidelity Image Generation With Fewer Labels
- 적은 양의 label 데이터 (<20%)만 가지고 이미지 생성
- Unsupervised / semi-supervised에서 여러 variant 모델을 제안했는데, S^4L pretraiend 모델을 사용해서 pseudo label을 사용하는건 어떨지 궁금함
